docs/upgrade-guides/node-to-express.mdx

```diff
@@ -0,0 +1,205 @@
+---
+description: Learn how to upgrade from the Clerk's Node SDK to the Express SDK.
+---
+
+# Upgrade from Clerk's Node SDK to the Express SDK
+
+## Install the Express SDK
+
+To upgrade from the Clerk Node SDK to the Clerk Express SDK, uninstall the Node SDK by running the following command:
+
+<CodeBlockTabs options={["npm", "yarn", "pnpm"]}>
+  ```sh {{ filename: 'terminal' }}
+  npm uninstall @clerk/clerk-sdk-node
+  ```
+
+  ```sh {{ filename: 'terminal' }}
+  yarn remove @clerk/clerk-sdk-node
+  ```
+
+  ```sh {{ filename: 'terminal' }}
+  pnpm remove @clerk/clerk-sdk-node
+  ```
+</CodeBlockTabs>
+
+Then, install the Express SDK by running the following command:
+
+<CodeBlockTabs options={["npm", "yarn", "pnpm"]}>
+  ```sh {{ filename: 'terminal' }}
+  npm install @clerk/express
+  ```
+
+  ```sh {{ filename: 'terminal' }}
+  yarn add @clerk/express
+  ```
+
+  ```sh {{ filename: 'terminal' }}
+  pnpm add @clerk/express
+  ```
+</CodeBlockTabs>
+
+## Migrate from `withAuth()`
+
+If you previously used `withAuth()` to wrap your route handlers, you can now use [`getAuth()`](/docs/references/express/overview#get-auth) with the Express SDK. Pass the request object to `getAuth()` to retrieve the authentication state within the route handler.
+
+`clerkMiddleware()` is required to be set in the middleware chain before this util is used, as shown in the following example.
+
+```ts {{ del: [1, [10, 13]], ins: [2, 8, [14, 17]] }}
```

Comment:
@bradlc and I had spoken about moving back to the diff syntax for this, just wanted to double check on whether this has happened or not yet. if it has, i think it would make this a little more maintainable to use the diff syntax!

Reply:
Ahh, this was already using `diff` before I updated 😆 will revert